docs/DeveloperGuide.md

```diff
@@ -262,29 +262,85 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
+NUS students who
+* want to make more connections with other students
+* want to find course mates
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
-
+**Value proposition**: Able to save contacts of other NUS students with NUS-specific information (ex. Year, major, modules taken by each person, housing, CCAs).
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …             | I want to …                                                                    | So that I can…                                                         |
+|:---------|--------------------|--------------------------------------------------------------------------------|------------------------------------------------------------------------|
+| `* * *`  | new user           | see usage instructions                                                         | refer to instructions when I forget how to use the App                 |
+|
```

Comment:
There is an extra row between each user story. Is it intended?

![image](https://github.com/user-attachments/assets/17b01d7c-3a3b-43e4-896e-3b75be6cae38)